src/objects.js

```diff
@@ -1,41 +1,57 @@
+const { arrayToCSVString } = require("./arrays");
+
 const createPerson = (name, age) => {
-  // your code here
+  return {
+    name: name,
+    age: age,
+  };
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
```

Comment:
This is a nice example where you used the square notation instead of the dot notation.
Since you didn't know which property you want to find in the object, you'll use square notation to make it dynamically.

If you tried to return object.property, you would get an error saying that the property doesn't exist.